### PR TITLE
Validate  - old format files Improved error output

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28627

## Description
Validate  - old format files Improved error output

## Must have
- [ ] Tests

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@amshamah419 Can you briefly review the RN here? 🙏

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 * Fixed an issue in the **format** command where unnecessary files were not skipped.
 * Fixed an issue in the **update-release-notes** command where the *text* argument was not respected in all cases.
 * Fixed an issue in the **validate** command where a warning about detailed description was given for unified or deprecated integrations.
+* Improved **validate** command error output on old format files.
```
Comment (amshamah419) [inline CHANGELOG.md:9]:
```suggestion
* Improved the error returned by the **validate** command when running on files using the old format.
```

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 2]
```diff
@@ -824,16 +824,19 @@ def test_validate_no_old_format__with_toversion(self):
...
             When:
-                - running validate_no_old_format on the file
+                - running validate_no_old_format on the files
             Then:
-                - return a False as the file is invalid
+                - return a False as the files are invalid
+                - assert the handle_error function is called for each file
         """
-        mocker.patch.object(BaseValidator, "handle_error", return_value="not-a-non-string")
+        temp = mocker.patch.object(BaseValidator, "handle_error", return_value="not-a-non-string")
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/validate/tests/validators_test.py:834]:
```suggestion
        handle_error_mock = mocker.patch.object(BaseValidator, "handle_error", return_value="not-a-non-string")
```

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 3]
```diff
@@ -824,16 +824,19 @@ def test_validate_no_old_format__with_toversion(self):
...
         validate_manager = ValidateManager()
-        old_format_files = {"demisto_sdk/tests/test_files/script-valid.yml"}
+        old_format_files = {"demisto_sdk/tests/test_files/script-valid.yml",
+                            "demisto_sdk/tests/test_files/integration-test.yml"}
         assert not validate_manager.validate_no_old_format(old_format_files)
+        assert temp.call_count == 2
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/validate/tests/validators_test.py:839]:
```suggestion
        assert handle_error_mock.call_count == 2
```

**Review by ShahafBenYakir — APPROVED** [review]
Nice solution for the unit tests

**Pull request merged**